sys/net/gnrc/network_layer/sixlowpan/frag/sfr/Kconfig

```diff
@@ -98,4 +98,54 @@ config GNRC_SIXLOWPAN_SFR_DG_RETRIES
     int "The maximum number of retries from scratch for a particular datagram (MaxDatagramRetries)"
     default 0
 
+menuconfig KCONFIG_USEMODULE_GNRC_SIXLOWPAN_FRAG_SFR_ECN_NETIF
+    bool "Configure SFR ECN based on netif queue"
```

Comment:
```suggestion
    bool "Configure SFR Explicit Congestion Notification based on netif queue"
```

Reply:
I'll add it up top to the UseECN config, as this is the first time in that menu that ECN is mentioned